Scripts/Python/ki/__init__.py

```diff
@@ -4315,6 +4313,18 @@ def BigKIProcessContentList(self, removeInboxStuff=False):
         for removeidx in removeList:
             del self.BKContentList[removeidx]
 
+        if self.BKFolderListOrder[self.BKFolderSelected] == xLocTools.FolderIDToFolderName(PtVaultStandardNodes.kInboxFolder):
```

Comment:
I'm not looking at the code right now, but I wonder if there is a way to compare integer folder types instead of the folder names?

Reply:
I'm not sure about that. Perhaps? I was doing what was consistent with other comparisons in the same method and throughout the KI code.